README.md

```diff
@@ -419,197 +745,512 @@ const myToolbar = {
+
+```js
+<DocumentView
+  overrideLongPressMenuBehavior={[Config.LongPressMenu.search]}
+/>
+```
 
-Defines menu items that should skip default behavior.
 ##### onLongPressMenuPress
 function, optional
 
-Defines what happens on long press menu press if it is passed in to `overrideLongPressMenuBehavior`
+This function is called if the pressed long press menu item is passed in to [`overrideLongPressMenuBehavior`](#overrideLongPressMenuBehavior)
 
 Parameters:
 
 Name | Type | Description
 --- | --- | ---
-longPressMenu | string | One of `Config.LongPressMenu` string constants
+longPressMenu | string | One of [Config.LongPressMenu](./src/Config/Config.js) constants, representing which item has been pressed
 longPressText | string | the selected text if pressed on text, empty otherwise
 
+```js
+<DocumentView
+  onLongPressMenuPress = {({longPressMenu, longPressText}) => {
+    console.log('Long press menu item', longPressMenu, 'has been pressed');
+    if (longPressText !== '') {
+      console.log('The selected text is', longPressText);
+    }
+  }}
+/>
+```
+
 ##### overrideBehavior
-array of `Config.Actions` string constants, optional
+array of string, optional, default to none
+
+Defines actions that should skip default behavior, such as external link click. Strings should be [Config.Actions](./src/Config/Config.js) constants.
+
+```js
+<DocumentView
+  overrideBehavior={[Config.Actions.linkPress]}
+/>
+```
 
-Defines actions that should skip default behavior, such as external link click.
 ##### onBehaviorActivated
 function, optional
 
-Defines what happens on certain behavior if it is passed in to `overrideBehavior`
+This function is called if the activated behavior is passed in to [`overrideBehavior`](#overrideBehavior)
 
 Parameters:
 
 Name | Type | Description
 --- | --- | ---
-action | string | One of `Config.Actions` string constants
+action | string | One of [Config.Actions](./src/Config/Config.js) constants, representing which action has been activated
 data | object | A JSON object that varies depending on the action
 
-data param table:
+Data param table:
 
 Action | Param
 --- | ---
-`Config.Actions.linkPress` | key: `url`, value: the link pressed
+[`Config.Actions.linkPress`](./src/Config/Config.js) | key: `url`, value: the link pressed
+
+```js
+<DocumentView
+  onBehaviorActivated = {({action, data}) => {
+    console.log('Activated action is', action);
+    if (action === Config.Actions.linkPress) {
+      console.log('The external link pressed is', data.url);
+    }
+  }}
+/>
+```
 
 ##### pageChangeOnTap
 bool, optional, default to true
+
+Defines whether page should change on tap when viewer is in horizontal viewing mode
+
+```js
+<DocumentView
+  pageChangeOnTap={true}
+/>
+```
+
 ##### useStylusAsPen
 bool, optional, default to true
 
-If true, stylus will act as a pen in pan mode, otherwise it will act as finger
+Defines whether stylus should act as a pen in pan mode. If false, it will act as a finger.
```

Comment:
```suggestion
Defines whether a stylus should act as a pen when in pan mode. If false, it will act as a finger.
```

Reply:
We just added better support for this in the latest iOS version. @Maxwell-Yang-PDFTron have you tested with it?